docs/SNAPTELD.md

```diff
@@ -57,38 +59,92 @@ $ $SNAP_PATH/bin/snapteld [global options] command [command options] [arguments.
 ```
 
 ## Examples
+
 ### Commands
 ```
-$SNAP_PATH/bin/snapteld
-$SNAP_PATH/bin/snapteld -log-level 4
-$SNAP_PATH/bin/snapteld -l 1 -t 2 -k <keyringPath>
-$SNAP_PATH/bin/snapteld -a $SNAP_PATH/plugins/
-$SNAP_PATH/bin/snapteld --version
+$ snapteld
+$ snapteld --version
+$ snapteld --log-level 4
+$ snapteld --auto-discover /opt/snap/plugins/
+$ snapteld --log-level 1 --plugin-trust 2 --keyring-paths /etc/snap/keyrings
 ```
 
-### Output
-```
-$ $SNAP_PATH/bin/snapteld -l 1 -t 0 --rest-auth
-```
+### Debug output
+By default, Snap daemon loads the configuration in `/etc/snap/snapteld.conf` and writes logs to `/var/log/snap/snapteld.log`. When debugging Snap issues, instead of a daemon, you can run it as a foreground process to review the logs directly:
+
+* shutdown `snap-telemetry` service and ensure there's no snap processes running:
+    RedHat 6/Ubuntu 14.04:
+    ```
+$ sevice snap-telemetry stop
+$ pgrep snap
+    ```
+    RedHat 7/Ubuntu 16.04:
+    ```
+$ systemctl stop snap-telemetry
+$ pgrep snap
+    ```
+
+* run snap with debug log `--log-level 1`, no log file `--log-path ''`, along with any other custom startup options (it will use the snapteld.conf file settings if an option is omited):
+    ```
+$ /usr/local/sbin/snapteld --log-level l 1 --log-path '' --plugin-trust 0
```

Comment:
This command: 
` /usr/local/sbin/snapteld --log-level l 1 --log-path '' --plugin-trust 0` 
should be without this single "l" letter like below: 
` /usr/local/sbin/snapteld --log-level 1 --log-path '' --plugin-trust 0` 

Also, please check is it fine in this place to skip the part "/usr/local/sbin/" in this command (just light suggestion)